packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/vsphere-users/add/vsphere-user-add.controller.js

```diff
@@ -29,18 +29,23 @@ export default class {
   }
 
   addUser() {
+    this.trackClick('add::confirm');
     this.loaders.add = true;
     this.DedicatedCloud.addUser(this.productId, this.newUser).then(
       () => {
-        this.goBack(this.$translate.instant('dedicatedCloud_users_add_start'));
+        return this.goBackWithTrackingPage({
+          message: this.$translate.instant('dedicatedCloud_users_add_start'),
+          trackingTag: 'add-success',
+        });
```

Comment:
suggestion: Since the function body has only one statement, `{ }` and `return` can be avoided.

Reply:
It's applied.